localstack/services/awslambda/provider.py

```diff
@@ -402,6 +404,11 @@ def _publish_version_from_existing_version(
         self.lambda_service.publish_version(new_version)
         state = lambda_stores[account_id][region]
         function = state.functions.get(function_name)
+        # TODO: re-evaluate data model to prevent this dirty hack just for bumping the revision id
```

Comment:
Yes, the intention was to have a transparent mechanism to change revision ids without manually doing it, but publishing versions is the perfect counterexample for that.

Reply:
I see. Commendable efforts as it works most of the time and makes it easier for these cases.